zaza/utilities/openstack.py

```diff
@@ -262,14 +263,16 @@ def get_undercloud_keystone_session(verify=None):
                                 verify=verify)
 
 
-def get_keystone_session_client(session):
+def get_keystone_session_client(session, api_version=None):
     """Return keystoneclient authenticated by keystone session.
 
     :param session: Keystone session object
     :type session: keystoneauth1.session.Session object
     :returns: Authenticated keystoneclient
     :rtype: keystoneclient.v3.Client object
     """
+    if api_version is not None:
```

Comment:
This looks wrong, if I explicitly ask for a v3 client I'll get a v2 one

Reply:
You're right, my head was tuned to "you will only ever use this option if you want something other than the default which is 3" when I wrote this.

Will fix :+1: